fms/models/gpt_bigcode.py

```diff
@@ -30,6 +30,8 @@ class GPTBigCodeConfig(ModelConfig):
     emb_dropout: float = 0.0
     multiquery_attn: bool = True
     ln_eps: float = 1e-5
+    linear_config: Optional[Mapping[str, Any]] = None  # pass as {"linear_type": str, <other kwargs>}
```

Comment:
We should test that this works properly with serialization and deserialization of the config

Reply:
I added `linear_config = {"linear_type": "torch_linear"}` and `unfuse_strategy=None` (_not_ parameterized) when testing the configs of GPTBigCode and Llama, so that we are leveraging all the tests in `ModelConfigTestSuite`.